.travis/check-strict-null-errors.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+snapshot=`cat .travis/check-strict-null-errors.snapshot` 
+calculated=`npx tsc --noEmit --strictNullChecks | node scripts/count-strict-null-check-errors.js` 
```

Comment:
@dominik-zeglen Can we save --noEmit and --strictNullChecks into a config file to make sure that linters will always have same config, no matter how configured?

Reply:
No, because webpack won't emit anything.
Also every build will fail because of `strictNullChecks`.